docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     build:
       context: .
       dockerfile: docker/pebble/Dockerfile
-    command: pebble -config /test/config/pebble-config.json -strict -dnsserver 10.30.50.3:8053
+    command: pebble -config /test/config/pebble-config.json -strict -dnsserver 10.30.50.3:8053 -defaultIPv6 "" -defaultIPv4 10.30.50.3
```

Comment:
I believe the -defaultIPv6/4 flags should be on the pebble-challtestsrv binary (not pebble).

Reply:
:laughing: You're 100% correct. I slapped this into a commit after I opened the PR and should have paid closer attention. Thanks for catching that! Fixed in 6adc5db